footprints/mgrs-to-county/geocode-grids/app.js

```diff
@@ -0,0 +1,240 @@
+fs = require('fs');
+geolib = require('geolib');
+mgrs = require('mgrs');
+pointInGeopolygon = require('point-in-geopolygon');
+pointInPolygon = require('point-in-polygon');
+proj4 = require('proj4');
+readline = require('readline');
+turf = require('@turf/turf')
+_ = require('underscore');
+
+stateCodes = require('./state-codes')
+
+
+// we must have a file for us to process
+if (process.argv.length < 3) {
```

Comment:
Can also use something like argparse for dealing with this stuff, pretty standard in python land (https://docs.python.org/3/library/argparse.html)